plugins/huaweicloud/src/main/java/org/eclipse/osc/orchestrator/plugin/huaweicloud/ObsOrchestratorStorage.java

```diff
@@ -1,49 +1,161 @@
 package org.eclipse.osc.orchestrator.plugin.huaweicloud;
 
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.util.HashSet;
+import java.util.List;
 import java.util.Set;
+import java.util.stream.Collectors;
+
+import com.obs.services.ObsClient;
+import com.obs.services.exception.ObsException;
+
+import com.obs.services.model.*;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.karaf.minho.boot.service.ConfigService;
 import org.apache.karaf.minho.boot.service.ServiceRegistry;
 import org.apache.karaf.minho.boot.spi.Service;
 import org.eclipse.osc.orchestrator.OrchestratorStorage;
 
+@Slf4j
 public class ObsOrchestratorStorage implements OrchestratorStorage, Service {
 
+    private static final String ACCESS_KEY = "HW_ACCESS_KEY";
+    private static final String SECRET_KEY = "HW_SECRET_KEY";
+    private static final String ENDPOINT = "ENDPOINT";
```

Comment:
```suggestion
    private static final String ENDPOINT = "HW_OBS_STORAGE_ENDPOINT";
```
For environment variables, we should make them more precise.